examples/rnn_classifer/model.py

```diff
@@ -0,0 +1,108 @@
+import os
+from urllib.request import urlretrieve
+import tarfile
+
+from dstoolbox.transformers import Padder2d
```

Comment:
Maybe the docs should mention that dstoolbox needs to be installed as well, since it's not part of the skorch requirements.